### PR TITLE
Tell GitHub's linguist to ignore scripts

### PR DESCRIPTION
**Pull request by LongTengDao**

### CODE REVIEW

**Review by jaceppo — APPROVED** [review]

**Comment by eksortso** [comment]
It's my first time looking at Linguist attributes. So based on [all I know](https://link.medium.com/2g6ikTb31gb), here's what this is.

If I'm understanding this correctly, this PR intends to declare to Github Linguist that the scripts are all vendored, so the project won't be confused for a regular Python project.

If so, then LGTM. The scripts technically aren't vendored, but the `toml-lang` folks could make new projects out of them, if that need ever arose, and could then vendor them in properly.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+/scripts/* linguist-vendored=true
```
Comment (pradyunsg) [inline .gitattributes:1]:
```suggestion
/scripts/* linguist-detectable=false
```

**Comment by LongTengDao** [comment]
Oh, you are right, `linguist-detectable` is be more suitable than `linguist-vendored`!

**Comment by pradyunsg** [comment]
Thanks @LongTengDao! ^.^

**Pull request merged**